### PR TITLE
Create tag component

### PR DESCRIPTION
**Pull request by lajanzen**
Created tag component to filter days. The filter function however has not been created yet :)

Check out the [Deployment](https://coolonia-tagcomponent-kusf6s2c.herokuapp.com/storybook/?path=/story/component-tag--today).

image

### CODE REVIEW

File: src/components/Tag/Tag.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import Tag from "./Tag";
+
+export default {
+  title: "Component/Tag",
+  component: Tag,
+};
+
+export const Today = (): JSX.Element => <Tag tagName="heute" value="today" />;
+export const Monday = (): JSX.Element => <Tag tagName="Mo" value="day1" />;
+export const Tuesday = (): JSX.Element => <Tag tagName="Di" value="day2" />;
```
Comment (julialoeschel) [inline src/components/Tag/Tag.stories.tsx:10]: great Idear to make the Tags as a radio inputs. Love it.

In storybook I see that the color chnages just one time to red. from the code I get, it is red(primary) when checked, and not red(seconday color) when unchecked. Maybe there is some issue, maybe it is just storybook.

**Review by julialoeschel — COMMENTED** [review]
great!

**Review by JulianSchumacher99 — APPROVED** [review]
🚀

File: src/components/Tag/Tag.stories.tsx (hunk 1 above)
Comment (lajanzen) [reply src/components/Tag/Tag.stories.tsx:10]: Great, thank you! :)
However, that is a normal behavior for a radio button. You can usually not "uncheck". I will consider implementing this later!

**Pull request merged**